From 4d2140da12a73546389b2f16a24d1d3589cb1151 Mon Sep 17 00:00:00 2001
From: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
Date: Thu, 14 Sep 2023 17:22:45 -0300
Subject: [PATCH] csm-replication: fix pull image error

This patch consists of adding imagePullSecrets with name
default-registry-key only in ServiceAccounts.
It's necessary because without it the system can't use the
images from the registry local.

Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
---
 helm/csm-replication/templates/controller.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/helm/csm-replication/templates/controller.yaml b/helm/csm-replication/templates/controller.yaml
index 23a9496..e2633e1 100644
--- a/helm/csm-replication/templates/controller.yaml
+++ b/helm/csm-replication/templates/controller.yaml
@@ -3,6 +3,8 @@ kind: ServiceAccount
 metadata:
   name: dell-replication-controller-sa
   namespace: {{ .Release.Namespace }}
+imagePullSecrets:
+  - name: default-registry-key
 secrets:
 - name: replication-secret
 ---
-- 
2.25.1
